tests/Jellyfin.Model.Tests/Dlna/StreamBuilderTests.cs

```diff
@@ -297,7 +297,7 @@ public async Task BuildVideoItemWithFirstExplicitStream(string deviceName, strin
         [InlineData("RokuSSPlus", "mp4-h264-ac3-aac-srt-2600k", PlayMethod.DirectPlay, (TranscodeReason)0, "Remux")] // #6450
         [InlineData("RokuSSPlus", "mp4-hevc-ac3-aac-srt-15200k", PlayMethod.DirectPlay, (TranscodeReason)0, "Remux")] // #6450
         // no streams
-        [InlineData("Chrome", "no-streams", PlayMethod.Transcode, TranscodeReason.VideoCodecNotSupported, "Transcode")] // #6450
+        [InlineData("Chrome", "no-streams", PlayMethod.DirectPlay, (TranscodeReason)0, "Remux")]
```

Comment:
Just a note: this has changed because the `null` input codec is now accepted by any codec list. Previously it was only accepted by a negative codec list.

Before:
https://github.com/jellyfin/jellyfin/blob/c67b78bc68317c66f3324230b2fc36918b2b9d70/MediaBrowser.Model/Dlna/ContainerProfile.cs#L38
After:
https://github.com/jellyfin/jellyfin/blob/d9c05036b8e96e928d6943644d2002573b2a466c/MediaBrowser.Model/Extensions/ContainerHelper.cs#L22

Reply:
Thanks for the fine. I think how it is right now is more consistent.

Reply:
Well, it depends on how to interpret the `null` codec. Previously it probably meant "unknown", which makes sense.

It might be a good idea to treat `null` as "not supported" by both, positive and negative, codec lists. But this also changes the previous logic.

I would not change any logic in "optimization" PR (i.e. refactoring).

Reply:
So revert back to old behavior and ignore it only on negative list?

Reply:
Yes, for now.